apps/web/src/components/BridgeForm.tsx

```diff
@@ -198,33 +225,37 @@ export default function BridgeForm({
   };
 
   const onTransferTokens = async (): Promise<void> => {
-    if (isSendingFromEthNetwork) {
-      // Revalidate entered amount after refetching EVM balance
-      const refetchedEvmBalance = await refetchEvmBalance();
-      if (
-        validateAmountInput(
+    setIsVerifyingTransaction(true);
+    const isBalanceSufficientVerified = await verifySufficientHWBalance(true);
+    if (isBalanceSufficientVerified) {
+      if (isSendingFromEthNetwork) {
+        // Revalidate entered amount after refetching EVM balance
+        const refetchedEvmBalance = await refetchEvmBalance();
+        if (
+          validateAmountInput(
+            amount,
+            new BigNumber(refetchedEvmBalance.data?.formatted ?? 0)
+          )
+        ) {
+          return;
```

Comment:
If this if condition is satisfied, button will not be enabled again
```suggestion
          setIsVerifyingTransaction(false);
          return;
```